browser-test/src/applicant_application_index.test.ts

```diff
@@ -76,6 +80,10 @@ describe('applicant program index page', () => {
       wantSubmittedPrograms: [primaryProgramName],
     })
 
+    // Check that the question repeated in the second program shows previously answered.
+    await applicantQuestions.clickApplyProgramButton(otherProgramName)
+    await validateScreenshot(page, 'question-shows-previously-answered')
```

Comment:
**issue (blocking):** This should be in its own test, not the "categorizes programs for draft and applied applications" one

While some of our tests are monolithic they typically are so with a common theme to reduce redundancy or to reduce redundant setup, which isn't a concern here.

Reply:
Sounds good - updated to separate into separate test.

Reply:
It's ok to keep it in the same file and describe and that lets you re-use the beforeAll setup which is good. 

It just should be it's own `it` test.

Reply:
It's ok to keep it in the same file and describe and that lets you re-use the beforeAll setup which is good. 

It just should be its own `it` test.

Reply:
Ohh makes sense! Updated!